<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUnitFormClass</class>
 <widget class="QDialog" name="AddUnitFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddUnitForm</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="14" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Choose unit type:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Enter unit morality:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="18" colspan="2">
     <widget class="QListWidget" name="UnitsListWidget"/>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="ToMenu">
      <property name="styleSheet">
       <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
      </property>
      <property name="text">
       <string>Go back</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Enter unit health:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QLineEdit" name="UnitArmor">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Enter unit name:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QLineEdit" name="UnitMorality">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="14" column="5">
     <widget class="QPushButton" name="ChangeItemsButton">
      <property name="text">
       <string>Change Items</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Enter min base power:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="13" column="4">
     <widget class="QLineEdit" name="UnitMoralityChanges">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QListWidget" name="UnitTypes"/>
    </item>
    <item row="17" column="4">
     <widget class="QLabel" name="ChoosedUnitTargetType">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="17" column="3">
     <widget class="QLabel" name="ChoosedUnitType">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="3" colspan="2">
     <widget class="QLineEdit" name="changeViability">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="16" column="4">
     <widget class="QListWidget" name="UnitTargetTypes"/>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Enter max base power:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QPushButton" name="DeleteUnitButton">
      <property name="text">
       <string>Delete Unit</string>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Choose unit priority target type:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLineEdit" name="UnitMaxPower">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Enter unit armor:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Units</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="UnitMinPower">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="UnitNameLineEdit"/>
    </item>
    <item row="12" column="4">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Enter morality changes rate:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="AddUnit">
      <property name="text">
       <string>Add Unit</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QCheckBox" name="fortificationTargetCheckBox">
      <property name="text">
       <string>Set fortification priority target</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QCheckBox" name="renovareArmorCheckBox">
      <property name="text">
       <string>Renovate armor durng battle?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
